src/App.tsx

```diff
@@ -3,165 +1,46 @@
 import './App.scss';
+import { NavLink, Outlet, useLocation } from 'react-router-dom';
+import classNames from 'classnames';
+import { HomePage } from './components/HomePage';
+
+export const App = () => {
+  const { pathname } = useLocation();
+  const NLClass = ({ isActive }: { isActive: boolean }) => {
+    return classNames('navbar-item', {
+      'has-background-grey-lighter': isActive,
+    });
+  };
+
+  return (
+    <>
+      <html className="has-navbar-fixed-top" />
```

Comment:
The `html` tag is not valid inside JSX. If you intended to set a class on the `html` element, this should be done in the `index.html` file or using a different approach in React.